packages/error-reporting/src/index.js

```diff
@@ -154,6 +154,16 @@ function Errors(initConfiguration) {
    * app.use(errors.koa);
    */
   this.koa = koa(this._client, this._config);
+
+  if (this._config.getReportUnhandledRejections()) {
+    var that = this;
+    process.on('unhandledRejection', function(reason) {
+      console.log('UnhandledPromiseRejectionWarning: ' +
+        'Unhandled promise rejection: ' + reason +
```

Comment:
It might be good to include either "google" or the name of the module in this log message. I'm not sure "error-reporting console" is specific enough on its own.